gulpfile.js

```diff
@@ -355,5 +355,5 @@ gulp.task('prod', function(callback) {
 
 // Runserver for development
 gulp.task('dev:runserver', function(callback) {
-    runSequence('dev', 'lint', 'connect', 'watch', callback);
+    runSequence('dev', 'connect', 'watch', callback);
```

Comment:
I think there is no need to remove `lint` from here and to add as a dependency of `connect`. If you are thinking something for future, let me know.
FYI: runSequence run the task in sequence. If you want to run tasks in parallel then put them inside brackets like this. `['task1', 'task2']`